kuksa_viss_client/__init__.py

```diff
@@ -63,6 +63,8 @@ def _sendReceiveMsg(self, req, timeout):
     def authorize(self, token=None, timeout = 2):
         if token == None:
             token = self.tokenfile
+        token = os.path.expanduser(token)
+        print("token is " + token)
```

Comment:
Why printing?

Reply:
maybe better to give a suggestion, so I can use this cool feature from github to fix it more quickly :)
```suggestion
```